sdk/storage/Azure.Storage.Common/tests/Shared/TestEventListener.cs

```diff
@@ -79,6 +97,10 @@ public static void LogEvent(EventWrittenEventArgs args)
 
             // Dump the message and category
             Trace.WriteLine(message, category);
+
+            // Add the message to event buffer
+            s_eventBuffer.Append(message);
```

Comment:
Do we check if this is null before using it?

Reply:
It shouldn't be null if that method is executing. If it is null, that would indicate a bug somewhere so I'd rather not add a check.

Reply:
Then add an Assert it's not null with a message that you need to call the setup method first.  Somebody's going to mess this up in the future and it'd be nice to help them out when they do.